python/ray/dashboard/modules/job/job_supervisor.py

```diff
@@ -101,8 +103,17 @@ def _configure_logger(self) -> None:
             f"jobs/supervisor-{self._job_id}.log",
         )
         os.makedirs(os.path.dirname(supervisor_log_file_name), exist_ok=True)
-        self._logger.addHandler(logging.StreamHandler())
-        self._logger.addHandler(logging.FileHandler(supervisor_log_file_name))
+        self._logger.addFilter(CoreContextFilter())
```

Comment:
My current concern is that Ray has too many loggers, and each logger is configured in a different place. If users want to enable structured logging, they need to configure multiple settings.

Reply:
There are only two settings: LoggingConfig for application logs and RAY_BACKEND_LOG_JSON for system logs (JobSupervisor is system log).